packages/next/src/build/utils.ts

```diff
@@ -1657,10 +1675,18 @@ export async function isPageStatic({
           )
         }
 
+        // A page supports partial prerendering if it is an app page and either
+        // the whole app has PPR enabled or this page has PPR enabled when we're
+        // in incremental mode.
+        isRoutePPREnabled =
+          routeModule.definition.kind === RouteKind.APP_PAGE &&
+          !isInterceptionRouteAppPath(page) &&
+          checkIsRoutePPREnabled(pprConfig, appConfig)
```

Comment:
I wonder if the interception check should be part of `checkIsRoutePPREnabled` so we don't accidentally forget it somewhere